### PR TITLE
Remove itwinui-css dependency

### PR DESCRIPTION
**Pull request by saskliutas**
Closes https://github.com/iTwin/presentation/issues/287
Closes https://github.com/iTwin/presentation/issues/253

### CODE REVIEW

File: packages/components/src/presentation-components/properties/NavigationPropertyTargetSelector.tsx [hunk 1]
```diff
@@ -89,19 +87,18 @@ export const NavigationPropertyTargetSelector = forwardRef<NavigationPropertyTar
...
         styles={{
-          control: () => ({ height: "27px" }),
-          container: () => ({ width: "auto" }),
-          valueContainer: () => ({ height: "27px", ["--_iui-select-padding-block"]: 0, ["--_iui-select-min-height"]: "var(--iui-component-height-small)" }),
-          menu: () => ({ position: "absolute", zIndex: 9999, width }),
-          menuList: (style: any) => ({ ...style, padding: 0 }),
-          option: () => ({ whiteSpace: "nowrap", width: "max-content", minWidth: "100%" }),
-          dropdownIndicator: () => ({ backgroundColor: "var(--iui-color-background)" }),
+          control: () => ({}),
+          container: () => ({}),
```
Comment (grigasp) [inline packages/components/src/presentation-components/properties/NavigationPropertyTargetSelector.tsx:91]: do they need to be specified if they're returning empty styles?

File: packages/components/src/test/instance-filter-builder/IntanceFilterBuilder.test.tsx [hunk 2]
```diff
@@ -48,89 +48,40 @@ describe("InstanceFilter", () => {
     sinon.restore();
   });
 
-  it("invokes 'onClassSelected' when non selected class is clicked", () => {
+  it("invokes 'onClassSelected' when non selected class is clicked", async () => {
```
Comment (grigasp) [inline packages/components/src/test/instance-filter-builder/IntanceFilterBuilder.test.tsx:51]: test names need to be updated

File: packages/components/src/presentation-components/properties/NavigationPropertyTargetSelector.tsx (hunk 1 above)
Comment (saskliutas) [reply packages/components/src/presentation-components/properties/NavigationPropertyTargetSelector.tsx:91]: Yes. These are for removing default styling.

File: packages/components/src/test/instance-filter-builder/IntanceFilterBuilder.test.tsx (hunk 2 above)
Comment (saskliutas) [reply packages/components/src/test/instance-filter-builder/IntanceFilterBuilder.test.tsx]: Fixed

**Review by grigasp — APPROVED** [review]

**Pull request merged**